api/types/types.go

```diff
@@ -75,21 +76,23 @@ var TransportPrometheusAnnotations = map[string]string{
 
 // Controller constants
 const (
-	ControllerDeploymentName     string = "skupper-service-controller"
-	ControllerComponentName      string = "service-controller"
-	ControllerContainerName      string = "service-controller"
-	ControllerServiceAccountName string = "skupper-service-controller"
-	ControllerRoleBindingName    string = "skupper-service-controller"
-	ControllerRoleName           string = "skupper-service-controller"
-	ControllerConfigPath         string = "/etc/messaging/"
-	ControllerServiceName        string = "skupper"
+	ControllerDeploymentName             string = "skupper-service-controller"
+	ControllerComponentName              string = "service-controller"
+	ControllerContainerName              string = "service-controller"
+	ControllerServiceAccountName         string = "skupper-service-controller"
+	ControllerRoleBindingName            string = "skupper-service-controller"
+	ControllerClusterRoleBindingNsFormat string = "skupper-service-controller-%s"
+	ControllerRoleName                   string = "skupper-service-controller"
+	ControllerClusterRoleName            string = "skupper-service-controller"
+	ControllerConfigPath                 string = "/etc/messaging/"
+	ControllerServiceName                string = "skupper"
 )
 
 var ControllerPolicyRule = []rbacv1.PolicyRule{
 	{
 		Verbs:     []string{"get", "list", "watch", "create", "update", "delete"},
 		APIGroups: []string{""},
-		Resources: []string{"services", "configmaps", "pods", "secrets"},
+		Resources: []string{"services", "configmaps", "pods", "pods/exec", "secrets"},
```

Comment:
Why does the controller need to exec into other pods?

Reply:
This can be removed now. I have exposed an API to validate policies, which CLI is using through the get command.
At some point I used that API from inside the controller, which does not make sense, so I probably left this behind.

I will remove and retest.

Reply:
Right now `ConnectorCreateSecretFromData` and `TokenClaimTemplateCreate` are using the `PolicyAPIClient` (which
uses the get command in order to verify policies from within the controller).
These VanClient methods are used by the CLI, by TokenManager and by LinkManager.

I will evaluate the impact of using the `ClusterPolicyValidator` component itself (which is meant for internal controller
 usage) and get rid of this extra resource permission.